packages/data-context/src/sources/migration/format.ts

```diff
@@ -15,8 +15,8 @@ export type FilePart = FilePartNoHighlight | FilePartHighlight
 
 export const supportFileRegexps = {
   e2e: {
-    beforeRegexp: 'cypress/\support\/(?<name>index)\.(?=[j|t]s[x]?)',
-    afterRegexp: 'cypress/\support\/(?<name>e2e)\.(?=[j|t]s[x]?)',
+    beforeRegexp: 'cypress[\\\\/]support[\\\\/](?<name>index)\.(?=[j|t]sx?)',
+    afterRegexp: 'cypress[\\\\/]support[\\\\/](?<name>e2e)\.(?=[j|t]sx?)',
```

Comment:
`[\\]` and `[\\\\]` should be identical character classes

```suggestion
    beforeRegexp: 'cypress[\\/]support[\\\\/](?<name>index)\.(?=[j|t]sx?)',
    afterRegexp: 'cypress[\\/]support[\\\\/](?<name>e2e)\.(?=[j|t]sx?)',
```

Reply:
Unfortunately no since this is used as a RegExp (so code `\\\\` => string `\\` => RegExp => `\` ).

It confused me too...